Tasks/VsTest/vstest.ts

```diff
@@ -1088,4 +1112,4 @@ function getTIALevel()
         return "file";
```

Comment:
can we make file as default. meaning if user doesn't give anything make it file. not related to ur change